.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -131,3 +131,4 @@ jobs:
             make $BUILD_OPTIONS target/sonic-$(GROUP_NAME).bin
           fi
         displayName: "Build sonic image"
+      - template: check-dirty-version.yml
```

Comment:
>template: check-dirty-version.yml [](http://example.com/codeflow?start=8&length=33)

Is it too late. If we detect dirty, we don't need to build at all.

Reply:
@qiluo-msft , the dirty is detected after build, during the build, it may not have the dirty issue. In most case 99%, no one changes the source code, so I think it is not late to check it after build.